tests/test_waku_lightpush.nim

```diff
@@ -4,15 +4,17 @@ import
   testutils/unittests,
   chronicles,
   chronos,
-  libp2p/crypto/crypto
+  libp2p/crypto/crypto,
+  std/options
```

Comment:
Nitpick: in general our imports are alphabetical, but we add `std` imports at the very top of the import list, followed by `testutils`